pkgs/servers/draupnir/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl common-updater-scripts nix coreutils jq
+
+set -euo pipefail
+
+echo "Getting versions..."
+latestVersion="$(curl -s "https://api.github.com/repos/Gnuxie/Draupnir/releases?per_page=1" | jq -r '.[0].tag_name | ltrimstr("v")')"
+echo " --> Latest version: ${latestVersion}"
+currentVersion=$(nix-instantiate --eval -E "with import ./. {}; draupnir.version or (lib.getVersion draupnir)" | tr -d '"')
+echo " --> Current version: ${currentVersion}"
+if [[ "$currentVersion" == "$latestVersion" ]]; then
+  echo "Draupnir is up-to-date: $currentVersion"
+  exit 0
+else
+  echo "We are out of date..."
+fi
+
+update-source-version draupnir 0 0000000000000000000000000000000000000000000000000000000000000000
+update-source-version draupnir "$latestVersion"
+
+# use patched source
+echo "Patching..."
+store_src="$(nix-build . -A draupnir.src --no-out-link)"
+
```

Comment:
```suggestion
```

Without this, the update script works fine for me.

Reply:
Honestly, not sure what this is even supposed to do.